orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -52,20 +52,14 @@
         return {
           description: entry.description,
           timeWorked: entry.time_worked.componentize(),
-          assignment: orchestraTasks.tasksById[entry.assignment]
+          task: orchestraTasks.tasksByAssignmentId[entry.assignment]
```

Comment:
is there a race condition here? Where the promise hasn't resolved yet, but entry.assignment is loaded from time entries?